src/views/statutory/PositionsList.vue

```diff
@@ -252,6 +271,28 @@ export default {
         }
       })
     },
+    deletePosition (position) {
+      this.$buefy.dialog.confirm({
+        title: 'Removing position',
+        message: 'Are you sure you want to remove the position for <b>' + position.name + '</b>? This action cannot be undone (easily).',
```

Comment:
Just say it cannot be undone, even if it's not true.

Reply:
Hahha yeah okay fair